animations-baker.js

```diff
@@ -534,6 +534,122 @@ const {CharsetEncoder} = require('three/examples/js/libs/mmdparser.js');
     });
     // console.log('got animations', animations);
 
+    const comboAnimationNames = [
```

Comment:
This code is completely unabstracted and unreadable.

Reply:
There are no comments or functions here, so it's not clear what this is supposed to do or why it would be correct.